published-202306-sanou-multiscale_glasso.qmd

```diff
@@ -772,14 +772,15 @@ ggplot(roc_dtf_erdos, aes(
     name = "Method",
     labels = c("GLasso", "MGLasso"),
     values = ghibli::ghibli_palette("MarnieMedium1")[5:6]
-  )
+  ) +
+  theme(text = element_text(size = 14))
```

Comment:
After decreasing the figure size as explained above there is no need for this change anymore

```suggestion
  ) 
```